index.js

```diff
@@ -0,0 +1,26 @@
+// import express (after npm install express)
+const express = require('express');
+
+// create new express app and save it as "app"
+const app = express();
+
+// server configuration
+const PORT = 3000;
+
+// JSON Data
+var data  = require('./package.json');
+
+// create a route for the app
+app.get('/', (req, res) => {
+  res.send('Hello World');
+});
+
+// create a route for the /info endpoint
+app.get('/info', (req, res) => {
+    res.send("{ serverName: '" + data.name + "', serverVersion: '" + data.version + "' }");
+});
+
+// make the server listen to requests
+app.listen(PORT, () => {
+  console.log(`Server running at: http://localhost:${PORT}/`);
+});
```

Comment:
Need a new line at the end of the file

Reply:
Thanks for pointing that out, just researched about importance of new line at end of file in unix :)